topo_processor/stac/item_factory.py

```diff
@@ -23,12 +23,14 @@ async def create_items(source_dir: str, data_type: DataType, target: str, temp_d
 
 
 async def process_item(item):
-    await metadata_loader_repo.add_metadata(item)
+    await metadata_loader_repo.load_metadata(item)
     if item.is_valid:
         await metadata_validator_repo.validate_metadata(item)
     if item.is_valid:
         await data_transformer_repo.transform_data(item)
     collection = get_collection(item.parent)
-    item.collection = collection
-    item.collection.metadata_path = os.path.join(item.parent, f"collection{collection.file_ext}")
-    collection.items.append(item)
+    if item.id in collection.items:
+        get_log().info("Duplicate Items", source_path=item.source_path)
```

Comment:
Is duplicate really only a info log? should this be a exception?

Reply:
changed it into an exception